allennlp/data/tokenizers/word_splitter.py

```diff
@@ -189,8 +189,10 @@ class BertBasicWordSplitter(WordSplitter):
     This is used to split a sentence into words.
     Then the ``BertTokenIndexer`` converts each word into wordpieces.
     """
-    def __init__(self, do_lower_case: bool = True) -> None:
-        self.basic_tokenizer = BertTokenizer(do_lower_case)
+    def __init__(self,
+                 do_lower_case: bool = True,
+                 never_split: Set[str] = ("[UNK]", "[SEP]", "[PAD]", "[CLS]", "[MASK]")) -> None:
```

Comment:
Should it be `never_split: Tuple[str]` instead of `never_split: Set[str]`?

Reply:
Yeah, you are right. My mistake.
I was thinking to make it `Set` because it should not be duplicate elements in it.
But I made to be Tuple here just to make it consistent with the original.
Also, since huggingface released their v0.5 pre-trained-bert, do you find there is any change we should make in Allennlp?